src/scss/constants.scss

```diff
@@ -33,6 +33,12 @@ $lighter-intermediate-grey: #3d3a3f;
 $light-grey: #292929;
 $light-grey--lightest: #acadb0;
 
+// @debt probably needs to be replaced with var(...), but it's used in a place
+// where var wasn't working
+$input-auto-fill-box-shadow: $secondary--dark;
+$input-auto-fill-box-shadow-timeline: #434343;
+$input-auto-fill-box-shadow-height: 30px;
+
```

Comment:
Delete. Not only replaced with var, but also with a `box-shadow--autofilled` function. Please don't use `$secondary-dark` there should be a comment after that color for suitable replacement. Similar to the gray `#434343` one, there are already existing ones at `:root`

Reply:
removed and replaced with var() 9bff7d8